tutorials/shaders/shader_reference/spatial_shader.rst

```diff
@@ -90,6 +94,59 @@ For visual examples of these render modes, see :ref:`Standard Material 3D and OR
 | **fog_disabled**              | Disable receiving depth-based or volumetric fog. Useful for ``blend_add`` materials like particles.  |
 +-------------------------------+------------------------------------------------------------------------------------------------------+
 
+Stencil modes
+-------------
+
+.. note::
+
+    Stencil support is experimental, use at your own risk.
+    We will try to not break compatibility as much as possible, 
+    but if significant flaws are found in the API, it may change
+    in the next minor version.
+
+Stencil operations are a set of operations that allow writing to
+an efficient buffer in an hardware-accelerated manner.
+This is generally used to mask in or out parts of the scene.
+
+Some of the most well-known uses are:
+
+- Outlines: Mask out the inner mesh that is being outlined to avoid inner outlines.
+- X-Ray: Display a mesh behind other objects.
+- Portals: Draw geometry that is normally "impossible" (non-Euclidian) by masking objects.
+
+.. note::
+
+    You can only read from stencil in the transparent pass.
+    Any attempt to read in the opaque pass will fail, as it's currently not supported behavior.
+
+    Also, stencil operations aren't yet supported in compositor effects.
```

Comment:
That's not true, stencil operations have been supported in compositor effects from the beginning. What isn't supporting is copying the main renderers stencil buffer into a custom texture

Reply:
I don't understand what this means. If stencils from the main renderer can't be used, then maybe it's technically correct to say they are supported, but i don't think it makes sense in the doc to say that you can use them..?

Reply:
@clayjohn poke